omics/cli/run_analyzer/__main__.py

```diff
@@ -345,21 +344,20 @@ def add_metrics(res, resources, pricing):
         # Get price for optimal static storage
         if store_max:
             capacity = get_static_storage_gib(store_max)
-        metrics["sizeMinimum"] = f"{capacity} GiB"
         gib_hrs = capacity * running / SECS_PER_HOUR
```

Comment:
I removed it from the `run` row so everything in the column is related to instance types instead of the mix of storage for runs and instances for tasks as it was previously. We could add in a new column for `recommendedMinimumStorage` or we already `storageReservedGiB`, `storageMaximumGiB`, `storageAverageGiB`

Open to suggestions on how you want to handle it.

Reply:
Because all rows are printed into one CSV table the number of columns must remain the same even if some values are empty for some row types.